src/backends/interflop-cancellation/interflop_cancellation.c

```diff
@@ -106,6 +106,27 @@ static inline double _noise_binary64(const int exp) {
     }                                                                          \
   })
 
+#define define_interflop_add_sub_vector(size, precision, ops, ope)             \
```

Comment:
For the backend cancellation, would you keep that ?
For the moment the backend is not really vectored because I didn't vectorize cancell function.
But multiplication and division didn't use this function, then they are vectored.

Reply:
keeping similar template for all backend if there is no perf penalty is desirable since it ease to get into the code